content-signup/src/content-signup.js

```diff
@@ -1,105 +1,121 @@
+      .onFormSubmit()
+      .create();
 }
 
-// Reviews each row to see who to send an email to.
-function onFormSubmit() {
-  let sheet = ss.getSheets()[0];
-  let rows = sheet.getDataRange().getValues();
-  
-  // ObjApp library uses the rangeToObjects() method to create a list of
-  // objects from the spreadsheet rows.
-  let recipients = ObjApp.rangeToObjects(rows);
-  for (recipient of recipients) {
-    // If an email was already sent, skip.
-    if (recipient[EMAIL_SENT_DATE])
-      continue;
+/**
+ * Sends a customized email for every response on a form.
+ *
+ * To see more of the onFormSubmit event, see:
+ * https://developers.google.com/apps-script/guides/triggers/events#form-submit
+ */
+function onFormSubmit(e) {
+  var responses = e.namedValues;
 
-    // Parse topics of interest into a list (since there are multiple items
-    // that are saved in the row as blob text).
-    let topics = [];
-    for (let topic in TOPIC_URLS) {
-      if (recipient[TOPICS].includes(topic.toUpperCase())) {
-        topics.push(topic);
-      }
-    }
+  // Get all the response values and store them in local variables.
+  // If the question title is a label, it can be accessed as an object field.
+  // If it has spaces or other characters, it can be accessed as a dictionary.
+  var timestamp = responses.Timestamp[0];
+  var email = responses['Email Address'][0].trim();
+  var name = responses.Name[0].trim();
+  var topicsString = responses.Topics[0].toLowerCase();
 
-    // If there is at least one topic selected, send email to the recipient,
-    // with this subject and email body.
-    if (topics.length > 0) {
-      let emailBody = createEmailBody(recipient, topics);
-      MailApp.sendEmail({
-        to: recipient[EMAIL],
-        subject: "Howdy",
-        htmlBody: emailBody,
-      });
+  // Parse topics of interest into a list (since there are multiple items
+  // that are saved in the row as blob of text).
+  var topics = [];
+  for (var topic in topicUrls) {
+    // indexOf searches for the topic in topicsString and returns a non-negative
+    // index if the topic is found, or it will return -1 if it's not found.
+    if (topicsString.indexOf(topic.toLowerCase()) != -1) {
+      topics.push(topic);
     }
-    
-    // Write the email sent date on the 'F' column.
-    let now = new Date();
-    sheet.getRange(`F${recipient.rowNum+1}`).setValue(now.toString());
+  }
 
-    // Logs activity.
-    Logger.log(JSON.stringify(recipient));
+  // If there is at least one topic selected, send an email to the recipient.
+  var status = '';
+  if (topics.length > 0) {
+    MailApp.sendEmail({
+      to: email,
+      subject: emailSubject,
+      htmlBody: createEmailBody(name, topics),
+    });
+    status = 'Sent';
   }
+  else {
+    status = 'No topics selected';
+  }
+
+  // Append the status on the spreadsheet to the responses' row.
+  var sheet = SpreadsheetApp.getActiveSheet();
+  var row = e.range.getRow();
```

Comment:
Optional: Use `sheet.getActiveRange().getRow()` instead, since auto-complete doesn't work well on the `e` event object